src/components/Premium/PremiumBrandCard.tsx

```diff
@@ -0,0 +1,113 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+import { PremiumCardLikeFillIcon, PremiumCardLikeIcon } from '../../assets/icon';
+import { BrandData } from '../../types/brand'; // BrandData import 추가
+import { postBrandData } from '../../utils/lib/brand';
+
+const PremiumBrandCard = (props: {brand: BrandData}) => {
+  
+  const { brand } = props;
+  
+  const [likeBtn, setLikeBtn] = useState(brand.likeTF);
+
+  const SendLikeRequest = async (brandId: number) => {
+    try {
+      const response = await postBrandData(brandId);
+      setLikeBtn(!likeBtn);
+      console.log(response.data);
```

Comment:
이제 response data 잘 들어갔는지 확인하는 과정 필요 없으면 (error는 어차피 밑에서 잡아줄테니!)
```suggestion
     postBrandData(brandId);
      setLikeBtn(!likeBtn);
```
쇽 지워도 되겠땅

Reply:
웅웅!!!! 지웠다 👍👍